src/NHibernate.ProxyGenerators/Default/DefaultProxyGenerator.cs

```diff
@@ -89,6 +89,12 @@ protected virtual CompilerResults CompileStaticProxyFactory(Configuration nhiber
 			foreach (var cls in nhibernateConfiguration.ClassMappings)
 			{
 				references.Add(cls.MappedClass.Assembly);
+				foreach (var mapReferenceAssemblyName in cls.MappedClass.Assembly.GetReferencedAssemblies())

```

Comment:
Sure, I'll get to that when I can!  thanks for merging the other PR's.  Is it sufficient to add a FluentNHibernate test which also tests this?